administrator-manual/en/accounts.rst

```diff
@@ -472,15 +472,20 @@ The default policy is :dfn:`strong`.
 Expiration
 ----------
 
-The  :index:`password expiration` is enabled by default to 6 months from the time when the password is set.
+The :index:`password expiration` is **NOT** enabled by default.
+If enabled, password lifetime is set to 6 months from the time when the password is set.
```

Comment:
If enabled, it is set according to `Password policies > The Maximum Password Age` value

Reply:
Ok for me